spec/karma.conf.js

```diff
@@ -90,7 +90,7 @@ module.exports = function (config) {
 		// - Opera
 		// - Safari (only Mac)
 		// - IE (only Windows)
-		browsers: ['Chrome1280x1024'],
+		browsers: ['Chrome1280x1024', 'SafariNative'],
```

Comment:
I don't think this should be added as a default here. Not all devs are running macOS, so it doesn't make sense to run Safari by default.
```suggestion
		browsers: ['Chrome1280x1024'],
```